lib/internal/Magento/Framework/Api/Code/Generator/ExtensionAttributesGenerator.php

```diff
@@ -41,6 +47,7 @@ class ExtensionAttributesGenerator extends \Magento\Framework\Code\Generator\Ent
      */
     public function __construct(
         \Magento\Framework\Api\ExtensionAttribute\Config $config,
+        \Magento\Framework\Reflection\TypeProcessor $typeProcessor,
```

Comment:
How about adding a `@TODO` comment to clean that mess up with the next backward compatibility breaking upgrade?